package-lock.json

```diff
@@ -2258,7 +2267,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
Here and elsehwere in many places, there is a https -> http change to package-lock.json. Could you please change these back to https?